roles/clevis/handlers/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Clean-initramfs
+  become: true
+  ansible.builtin.command: "update-initramfs -u -k all"
+  changed_when: false
+
+- name: "Restart {{ clevis_service_name }} service"
+  become: true
+  ansible.builtin.service:
+    name: "{{ clevis_service_name }}"
+    daemon_reload: true
+    enabled: true
+
+- name: Chain tang pin
+  become: true
+  ansible.builtin.command: |-
+     clevis luks bind -s 2 -f -d {{ crypt_root_disk }} tang '{ "url": "http://{{ tang_server_ip }}:{{ tang_server_port }}", "adv": "/tmp/adv.jws" }' {{ initial_luks_password }}
```

Comment:
Use ansible.builtin.tempfile

Introduce a new parameter tang_server_url as well as tang_server_protocol to build this URL.